using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+# Using the Portaal taak Plugin
+
+The Portaal Taak plugin is used to generate "taak" (task) objects within the Objecten API for a BPMN 
+user task. As a result, such tasks can be completed via external applications, such as the NL Portal. This feature 
+empowers Valtimo to assign tasks to other systems that are more adept at performing them. Furthermore, it enables users 
```

Comment:
```suggestion
empowers Valtimo to assign tasks to other systems that are more adept at performing them in a different context. Furthermore, it enables users 
```

I think the line as it is right now makes it seem like Valtimo isn't good for completing tasks. Not sure if that's the message we want to convey...

Reply:
I do agree with the issue you have with it, but not the suggestion. I don't think it's the context that is different. I change the text, please check if this is ok.